test/producers/remediation_updates_test.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class RemediationUpdatesTest < ActiveSupport::TestCase
+  setup do
+    @issue_id = 'ssg:rhel7|short_profile_ref_id|rule_ref_id'
+  end
+
+  test 'handles missing kafka config' do
+    assert_nil RemediationUpdates.deliver(
+      host_id: '0001', issue_ids: [@issue_id]
+    )
+  end
+
+  test 'delivers messages to the payload tracker topic' do
+    kafka = mock('kafka')
+    RemediationUpdates.stubs(:kafka).returns(kafka)
+    kafka.expects(:deliver_message)
+         .with(anything, topic: 'platform.remediation-updates.compliance')
```

Comment:
Seems silly to test the topic with a constant, but it reduces the brittleness of the test:
```suggestion
         .with(anything, topic: RemediationUpdates::TOPIC)
```